app/models/granule.rb

```diff
@@ -101,7 +104,7 @@ def self.create_granule(granule, current_user, granule_info)
       granule_record_data_list.push(granule_record_data)
     end
     granule_record.campaign = ApplicationController.helpers.clean_up_campaign(granule_record.campaign_from_record_data)
-    granule_record.save
```

Comment:
This line produced an empty review, so it is removed. Saving will be done in the caller.